.github/actions/prepare-api-variables/action.yml

```
@@ -73,6 +77,19 @@ runs:
           exit 0
         fi
 
+        if [[ "$MODE" == 'from-version' ]]; then
+          VERSION_SLUG=$(echo "${VERSION/./}")
```

Just style, could be simplified:
```suggestion
          VERSION_SLUG="${VERSION/./}"
```